rpcserver.go

```diff
@@ -2955,12 +2955,11 @@ func (r *rpcServer) GetInfo(_ context.Context,
 	}
 
 	network := lncfg.NormalizeNetwork(r.cfg.ActiveNetParams.Name)
-	activeChains := make([]*lnrpc.Chain, r.cfg.registeredChains.NumActiveChains())
-	for i, chain := range r.cfg.registeredChains.ActiveChains() {
-		activeChains[i] = &lnrpc.Chain{
-			Chain:   chain.String(),
+	activeChains := []*lnrpc.Chain{
+		{
+			Chain:   bitcoinChainName,
 			Network: network,
-		}
+		},
 	}
```

Comment:
I'm not sure what the process is for deprecated values but do we need a TODO to remove this when we actually strip this value from a future GetInfoResponse?